boards/cats_rev1Pro/Core/Src/tasks/task_init.c

```diff
@@ -339,6 +343,13 @@ static void init_baro() {
   osDelay(10);
 }
 
+static void init_magneto() {
+  spi_init(MAG.spi);
+  mmc5983ma_selftest(&MAG);
```

Comment:
Should we do something with this self test? Otherwise we can remove it

Reply:
Lets remove it then